docs/processors.md

```diff
@@ -99,22 +99,29 @@ The CiviCRM Contribution provides a way to use a Caldera Forms calculation field
 
 ### Memberships
 
-The CiviCRM Membership provides the ability to register and renew memberships. By using this processor you can renew existing memberships as is done using a contribution page, or create new memberships at each renewal. 
+The CiviCRM Membership processor provides the ability to register and renew memberships. By using this processor you can renew existing memberships as is done using a contribution page, or create new memberships at each renewal. If multiple memerships are provided, use one processor per membership. For free memberships the processor can work on it's own, for paid events use alongside the Line Item and Order processors. 
```

Comment:
Not sure what `, for paid events use alongside the Line Item and Order processors. ` is trying to explain.   Do we want to say that the Membership Processor can be used along with the Event Processor as long as you make use of Line Items and the Order processor?

Reply:
This means that each participant or membership processor needs a line item if it's a paid. So say if there were 2 event processors and 1 membership processors means 3 line item processors to then map into the order processor. If that makes any sense.